std/debug.zig

```diff
@@ -1874,9 +1878,10 @@ fn getLineNumberInfoMacOs(di: *DebugInfo, symbol: MachoSymbol, target_address: u
     return error.MissingDebugInfo;
 }
 
-fn getLineNumberInfoDwarf(di: *DwarfInfo, compile_unit: CompileUnit, target_address: usize) !LineInfo {
-    const compile_unit_cwd = try compile_unit.die.getAttrString(di, DW.AT_comp_dir);
-    const line_info_offset = try compile_unit.die.getAttrSecOffset(DW.AT_stmt_list);
+fn getLineNumberInfoDwarf(di: *DwarfInfo, compile_unit_die: Die, target_address: usize) !LineInfo {
+    const compile_unit_cwd = try compile_unit_die.getAttrString(di, di.allocator, DW.AT_comp_dir);
+    defer di.allocator.free(compile_unit_cwd);
```

Comment:
TODO: this `defer` may be the cause of andrew's issue: I noticed the path in his short traceback is missing a leading `/`. Perhaps it got freed and set to `0xaa` by `compile_unit_cwd` getting collected.